Support other platforms

trombik opened the pull request:
#3 is not yet fixed, though.

Commit:
initial support of Ubuntu

Commit:
initial support of OpenBSD

Commit:
update meta

Commit:
s/nfsd_exports_file/__nfsd_exports_file/

Commit:
add a handler to debug build in Jenkins

Commit:
add a comment why `rc_stop` is `NO`

it is deliverately disabled

Commit:
try `retries`

Commit:
fix `until`

Commit:
document `retries`

in Jenkins job, the handler fails a few times, succeeds after second or
third try.

trombik commented:
inconsistent variable names will be fixed in other branch.

mitsururike commented on tasks/configure-OpenBSD.yml:
```diff
@@ -0,0 +1,68 @@
+  # automagic.
+  # https://github.com/openbsd/src/commit/9217ca7aa6c2a8a0f0d5bd8516dbc7273e3686d2
+  #
+  # notify: Restart mountd
```
s/Restart mountd/Restart mountd OpenBSD/ ?
this comment should be written in `handlers/main.yml` as well where who wants to use it will read.

trombik replied:
correct

mitsururike reviewed (commented):
name of variables and facts should be nfs_server_*.
nfs_server, nfsd_server and nfsd_ are mixed.

Commit:
use `Restart mountd OpenBSD`

and put the comment in the right place.
https://github.com/reallyenglish/ansible-role-nfs-server/pull/4#discussion_r121040226

trombik commented:
the variable names are fixed in: 

https://github.com/reallyenglish/ansible-role-nfs-server/tree/tidy_up_and_readme

Commit:
fix wrong merge

Commit:
switch to master

the branch has gone
